documentation/asciidoc/topics/proc_creating_grafana_datasources.adoc

```diff
@@ -0,0 +1,65 @@
+[id='creating-grafana-dashboards_{context}']
+= Creating Grafana Data Sources
+Set up Prometheus as a data source so you can visualize {brandname} metrics in Grafana dashboards.
+
+.Prerequisites
+
+* Create a Prometheus service monitor for your {brandname} cluster.
+* Install the Grafana operator in your {brandname} cluster namespace.
+* Deploy a `Grafana` CR in your {brandname} cluster namespace.
+
+.Procedure
+
+. Create a service account for the Grafana data source.
++
+[source,options="nowrap",subs=attributes+]
+----
+include::yaml/prometheus_service_account.yaml[]
+----
++
+.. Apply the service account.
++
+[source,options="nowrap",subs=attributes+]
+----
+$ {oc_apply_cr} service-account.yaml
+----
++
+.. Assign the `cluster-monitoring-view` role to the service account.
++
+[source,options="nowrap",subs=attributes+]
+----
+$ oc adm policy add-cluster-role-to-user cluster-monitoring-view -z monitoring-account
+----
++
+. Export required system details to environment variables.
+.. Retrieve an authentication token for the service account.
++
+[source,options="nowrap",subs=attributes+]
+----
+$ export BEARER_TOKEN=$(oc serviceaccounts get-token monitoring-account)
+----
++
+.. Retrieve the URL for the Thanos Querier.
```

Comment:
I'm not sure why we're instructing users todo anything with Thanos. This is for multi-cluster aggregation of prometheus metrics. AFAIK this shouldn't be required for monitoring on a single k8 cluster.

Reply:
Thanks, yeah, this was in the notes I got earlier and just haven't dropped them yet. Will do that now.